### DISCUSSION TITLE
Please define "legacy account"

### DISCUSSION THREAD

**Discussion by The-Futurist** [community/community#21724] [New to GitHub]
<p>Can GitHub please accurately define what a legacy account is and what characteristics it has that makes it different to other non-lgacy account options.</p>
<p>Is there just one type of legacy account?</p>
<p>Can one visit their account page (e.g. our organization settings) and “convert” a legacy account to some other type of account or must we close the account and open a new one? If so might we lose our repos by mistake?</p>

**Comment by seveas** [comment, marked as answer]
<p>Hi <a class="mention" href="/korporal">@korporal</a>,</p>
<p>Legacy accounts are accounts that are billed per-repository instead of the per-user plans that are current, see <a href="https://github.com/pricing" rel="nofollow noopener">https://github.com/pricing</a> for the current plans and <a href="https://github.blog/2016-05-11-introducing-unlimited-private-repositories/" rel="nofollow noopener">https://github.blog/2016-05-11-introducing-unlimited-private-repositories/</a> for the announcement of the new pricing plans.</p>
<p>If you have an account on an older plan, you can change to a new plan without having to close the account.</p>
